src/Init/Init.c

```diff
@@ -107,54 +110,17 @@ Init_setup_pmc(void)
 static void
 Init_setup_pio(void)
 {
-    Pmc_enablePeripheralClk(Pmc_PeripheralId_PioD);
```

Comment:
This is empty now

Reply:
Yeah, it is. No initialization of PIO is required so far. Nevertheless, I have decided to keep this empty function in case of initialization is needed, then it will be obvious where to put it. If our standard discourages keeping empty function, I will delete it.